Makefile.standard_app

```diff
@@ -0,0 +1,160 @@
+    DEFINES += HAVE_PRINTF
+    ifeq ($(TARGET_NAME),TARGET_NANOS)
+        DEFINES += PRINTF=screen_printf
+    else
+        DEFINES += PRINTF=mcu_usb_printf
+    endif
+else
+        DEFINES += PRINTF\(...\)=
+endif
+
+#####################################################################
+#                        IO SEPROXY BUFFER SIZE                     #
+#####################################################################
+ifneq ($(DISABLE_DEFAULT_IO_SEPROXY_BUFFER_SIZE), 1)
+    ifeq ($(TARGET_NAME), TARGET_NANOS)
+        DEFINES += IO_SEPROXYHAL_BUFFER_SIZE_B=128
+    else
+        DEFINES += IO_SEPROXYHAL_BUFFER_SIZE_B=300
+    endif
+endif
+
+#####################################################################
+#                          STANDARD DEFINES                         #
+#####################################################################
+DEFINES += $(DEFINES_LIB)
+DEFINES += APPNAME=\"$(APPNAME)\"
+DEFINES += APPVERSION=\"$(APPVERSION)\"
+DEFINES += MAJOR_VERSION=$(APPVERSION_M) MINOR_VERSION=$(APPVERSION_N) PATCH_VERSION=$(APPVERSION_P)
+DEFINES += OS_IO_SEPROXYHAL
+
+ifeq ($(DISABLE_STANDARD_APP_DEFINES), 1)
+    DISABLE_STANDARD_SNPRINTF = 1
+    DISABLE_STANDARD_USB = 1
+    DISABLE_STANDARD_BLE = 1
+    DISABLE_STANDARD_WEBUSB = 1
+    DISABLE_STANDARD_BAGL_UX_FLOW = 1
+endif
+
+ifneq ($(DISABLE_STANDARD_SNPRINTF), 1)
+    DEFINES += HAVE_SPRINTF HAVE_SNPRINTF_FORMAT_U
+endif
+
+ifneq ($(DISABLE_STANDARD_USB), 1)
+    DEFINES += HAVE_IO_USB HAVE_L4_USBLIB IO_USB_MAX_ENDPOINTS=6 IO_HID_EP_LENGTH=64 HAVE_USB_APDU
+    DEFINES += USB_SEGMENT_SIZE=64
+endif
+
+ifneq ($(DISABLE_STANDARD_BLE), 1)
+    DEFINES += BLE_SEGMENT_SIZE=32
+endif
+
+ifneq ($(DISABLE_STANDARD_WEBUSB), 1)
+    DEFINES += HAVE_WEBUSB WEBUSB_URL_SIZE_B=0 WEBUSB_URL=""
+endif
+
+ifneq ($(DISABLE_STANDARD_BAGL_UX_FLOW), 1)
+ifneq ($(TARGET_NAME), TARGET_STAX)
+    DEFINES += HAVE_UX_FLOW
+endif
+endif
+
+ifneq ($(DISABLE_STANDARD_APP_FILES), 1)
+SDK_SOURCE_PATH += lib_standard_app
+endif
+
+#####################################################################
+#                              NBGL                                 #
+#####################################################################
+ifeq ($(ENABLE_NBGL_QRCODE), 1)
+ifeq ($(TARGET_NAME), TARGET_STAX)
+    DEFINES += NBGL_QRCODE
+endif
+endif
+
+ifeq ($(ENABLE_NBGL_KEYBOARD), 1)
+ifeq ($(TARGET_NAME), TARGET_STAX)
+    DEFINES += NBGL_KEYBOARD
+endif
+endif
+
+ifeq ($(ENABLE_NBGL_KEYPAD), 1)
+ifeq ($(TARGET_NAME), TARGET_STAX)
+    DEFINES += NBGL_KEYPAD
+endif
+endif
+
+CUSTOM_APP_FLAGS ?= 0x000
+
+APP_FLAGS = $(shell echo $$(( $(STANDARD_APP_FLAGS) + $(CUSTOM_APP_FLAGS) )) )
+
+APP_LOAD_PARAMS += --appFlags $(APP_FLAGS)
+
+APP_LOAD_PARAMS += $(COMMON_LOAD_PARAMS)
+
+CC      := $(CLANGPATH)clang
+AS      := $(GCCPATH)arm-none-eabi-gcc
+LD      := $(GCCPATH)arm-none-eabi-gcc
+LDLIBS  += -lm -lgcc -lc
+
+include $(BOLOS_SDK)/Makefile.glyphs
+
+STANDARD_APP_FLAGS = 0x000
```

Comment:
To be put near CUSTOM_APP_FLAGS with a comment that one can be personnalized and the other not?